constants/PROTOCOL_VERSION

```diff
@@ -0,0 +1 @@
+757
```

Comment:
Because I wanted to create a "one place for all constants" so we don't forget to update constants (protocol version number in initial_handlers, version string "Feather 1.18.1" in initial_handlers, anvil format number in base/anvil/region, and other constants in different places), but I completely forgot that I could create a constants.rs file.